specs/latest/2.0/index.html

```diff
@@ -415,6 +415,7 @@ <h2 class="no-toc">Table of contents</h2>
 typedef (Int32Array or sequence&lt;GLint&gt;) ClearBufferIVSource;
 typedef (Uint32Array or sequence&lt;GLuint&gt;) ClearBufferUIVSource;
 typedef (Float32Array or sequence&lt;GLfloat&gt;) ClearBufferFVSource;
+typedef (ArrayBuffer or SharedArrayBuffer) ByteBuffer;
```

Comment:
Note: the typedef can be renamed. Since it's used for bufferData, tex{Sub}Image, uniform*, etc., perhaps it could be renamed to ScalarDataSource or something similar.